tideways/usr/local/share/env/40-stack

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+export TIDEWAYS_HOSTNAME=${TIDEWAYS_HOSTNAME:-tideways-daemon}
```

Comment:
how about `WEB_HOST` as the default?

Reply:
WEB_HOST isn't normally on this container, but we could add it

Reply:
ah right, nvm, I wasn't thinking it was a separate image